Cython/Compiler/ExprNodes.py

```diff
@@ -11513,6 +11533,7 @@ def dispose(temp):
 
         dispose(shapes_temp)
         dispose(format_temp)
+        dispose(format_ptr_temp)
```

Comment:
```suggestion
        code.funcstate.release_temp(format_ptr_temp)
```

The `format_ptr_temp` isn't a `PyObject*` so doesn't want the `code.put_decref_clear(temp, py_object_type)` part of `dispose`

Reply:
Ah great thanks for the fix.